app/scripts/directives/configEditor.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('configEditor', function(APIService, DataService) {
+    return {
+      restrict: 'E',
+      scope: {
+        mobileClient: '<',
+      },
+      templateUrl: 'views/config-editor.html',
+      controller: function($scope) {
+        $scope.context = {namespace: _.get($scope, "mobileClient.metadata.namespace")};
+        //keep list of active $scope.services upto date
+        DataService.watch(
+          APIService.getPreferredVersion('serviceinstances'), 
+          $scope.context, 
+          function (serviceinstances){
+            $scope.services = [];
+            _.each(serviceinstances.by("metadata.name"), function(service, serviceName){
+              if(_.get(service, "metadata.labels.mobile") !== "enabled") {
```

Comment:
As we soon to remove labeling the service instance, does this make sense? I think we should be just looking for the configmaps labelled mobile enabled?

Reply:
I have to scan through services first, to ignore any excluded ones, so I have moved the check for this label to the configmap watch function, and removed it from here.